npm can no longer install rollbar 2.0 when git is unavailable

Before rollbar 2.0.0, `npm` could install the module with no problem in a stripped-down Alpine-based Docker container. However, now that there's a dependency on a module in GitHub, this fails:

```console
$  docker run -it node:6-alpine ash
/ # npm --loglevel error install rollbar@2.0.0
npm ERR! git clone --template=/root/.npm/_git-remotes/_templates --mirror git@github.com:rollbar/console-polyfill.git /root/.npm/_git-remotes/git-github-com-rollbar-console-polyfill-git-eab6ff9d2b7597fc2f259baa18100556bdb94dbe-77dcd1e0: undefined
npm ERR! git clone --template=/root/.npm/_git-remotes/_templates --mirror git@github.com:rollbar/console-polyfill.git /root/.npm/_git-remotes/git-github-com-rollbar-console-polyfill-git-eab6ff9d2b7597fc2f259baa18100556bdb94dbe-77dcd1e0: undefined
npm ERR! Linux 4.9.27-moby
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "--loglevel" "error" "install" "rollbar@2.0.0"
npm ERR! node v6.9.4
npm ERR! npm  v3.10.10
npm ERR! code ENOGIT

npm ERR! not found: git
npm ERR! 
npm ERR! Failed using git.
npm ERR! This is most likely not a problem with npm itself.
npm ERR! Please check if you have git installed and in your PATH.

npm ERR! Please include the following file with any support request:
npm ERR!     /npm-debug.log
```

The offending module is `console-polyfill` at https://github.com/rollbar/rollbar.js/blob/master/package.json#L12

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
   "main": "src/server/rollbar.js",
   "browser": "dist/rollbar.umd.min.js",
   "dependencies": {
-    "console-polyfill": "rollbar/console-polyfill#eab6ff9d2b7597fc2f259baa18100556bdb94dbe",
+    "console-polyfill": "0.3.0",
     "error-stack-parser": "1.3.3",
     "extend": "3.0.0",
     "async": "~1.2.1",

